docs/integrations/saml/okta.md

```diff
@@ -183,20 +183,15 @@ _sourceCategory = "okta" "user.mfa.factor.deactivate"
 ```
 
 
-
-## Configuring Okta Log Collection for Non-FedRamp Deployments
+## Configuring Okta Log Collection
 
 Use the [new Cloud to Cloud Integration for Okta](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source.md) to create the source and use the same source category while installing the app.
 
-The sections below are available for FedRamp Sumo Logic deployments to meet FedRamp's specific compliance requirements. If you're using the Sumo Logic FedRamp deployment, you must use the sections below to configure collection for this app.
-
-
-## Configuring Okta Log Collection for FedRamp Deployments
-
 :::note 
-Okta Log Collection configuration via SumoJanus is no longer applicable; instead, we offer Okta collection via hosted source for Fed deployment. Follow the instructions [here](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source/). The steps must be completed in the order they are presented.
+The Okta Log Collection configuration via SumoJanus is no longer applicable and deprecated. We recommend switching to [Cloud-to-Cloud integration](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source/) to configure the Okta Log collection. The steps must be completed in the order they are presented.
```

Comment:
```suggestion
The Okta Log Collection configuration via SumoJanus is no longer applicable and deprecated. We recommend switching to [Cloud-to-Cloud integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/okta-source) to configure the Okta Log collection. The steps must be completed in the order they are presented.
```

Reply:
As per the Contributor Guide, it's recommended to use full links. Let me know what needs to be done.
<img width="1369" alt="Screenshot 2023-01-20 at 11 56 35 AM" src="https://user-images.githubusercontent.com/116072777/213631615-b2864e04-d499-4a72-868b-92aac088f60c.png">

Reply:
That's for Release Notes links, because they're published on RSS feed. For intra-doc links, needs to be relative.

Reply:
Thanks @kimsauce. Could you please add this information to Guide?

Reply:
yes will do! added a note in Marie's recs.